app/javascript/components/batch_update/components/App.js

```diff
@@ -288,6 +305,22 @@ class App extends React.Component {
       })
   }
 
+  onAddMinPrice(price) {
+    if (price === null) return this.setState({ minPrice: null })
+    let newValue = parseInt(price)
+    if (!isNaN(newValue)) {
+      this.setState({ minPrice: newValue })
+    }
+  }
+
+  onAddMaxPrice(price) {
+    if (price === null) return this.setState({ maxPrice: null })
+    let newValue = parseInt(price)
+    if (!isNaN(newValue)) {
+      this.setState({ maxPrice: newValue })
+    }
+  }
+
```

Comment:
Let's change these to use the default state updaters below (that's what we use for single-valued fields like partner, before-date, after-date, ...)

The add/remove state updaters are only for fields that are multi-valued, like gene, tag, artist.

That also implies a UI change in that once a min-price or max-price is entered, the affordance for entering it again should go away.

Reply:
Gotcha, that makes sense!